llama_stack/apis/eval/eval.py

```diff
@@ -76,7 +74,7 @@ class EvaluateResponse(BaseModel):
 
 
 class Eval(Protocol):
-    @webmethod(route="/eval/run-eval", method="POST")
+    @webmethod(route="/eval/run", method="POST")
```

Comment:
Not sure how far do we want to go to be idiomatic?

should we have `/evals/tasks` that returns the list of task_ids? Then you would do `POST /evals/tasks/{task_id}/jobs` to create a new job?

And then use `/evals/tasks/{task_id}/jobs/{job_id}` for CRUD operations?